src/pl/plpython/plpy_main.c

```diff
@@ -80,6 +82,24 @@ cancel_pending_hook_type prev_cancel_pending_hook;
 void PLy_handle_cancel_interrupt(void);
 
 bool PLy_enter_python_intepreter = false;
+bool PLy_path_added = false;
+
+/* GUC variables */
+#if PY_MAJOR_VERSION >= 3
+char *plpython3_path = NULL;
+
+bool 
+plpython3_check_python_path(char **newval, void **extra, GucSource source) {
+	if (PLy_path_added)
+	{
+		GUC_check_errmsg("SET PYTHONPATH for plpython3 can only set once in one session");
+		return false;
+	}
+	if (strcmp(*newval, "") != 0)
```

Comment:
if we have this check, then the `PLy_path_added` is not needed. Since we can always check if the `plpython3_path` is `NULL`. Right?

Reply:
not for sure we need to make the default value empty string otherwise will cause the error (definecostumstring can not default NULL).